grpc-bridge-example/src/main/java/dev/resteasy/greet/GeneralGreeting.java

```diff
@@ -0,0 +1,25 @@
+package dev.resteasy.greet;
```

Comment:
We should name the package something like `dev.resteasy.example.grpc.greet`. While it's highly unlikely, we should try to avoid package splitting.

Reply:
Done